packages/ra-ui-materialui/src/input/SelectArrayInput.tsx

```diff
@@ -253,6 +254,7 @@ export const SelectArrayInput = (props: SelectArrayInputProps) => {
                 <Select
                     autoWidth
                     labelId={`${label}-outlined-label`}
+                    input={<OutlinedInput label={label} />}
```

Comment:
I'm afraid this will only work with the outlined variant. How about other variants?